src/patternfly/demos/Panel/Panel.md

```diff
@@ -0,0 +1,83 @@
+---
+id: Panel
+section: components
+---
+
+## Demos
+
+### With a menu
+
+```hbs
+{{#> panel}}
```

Comment:
super duper nit, would you mind updating this demo to the riased variant? I imagine that's the most common use case. 

```suggestion
{{#> panel panel--modifier="pf-m-raised"}}
```